File: Stories2/app/src/main/java/com/tecwec/stories/MainActivity.java
package com.tecwec.stories;

import androidx.appcompat.app.AppCompatActivity;
import androidx.recyclerview.widget.LinearLayoutManager;
import androidx.recyclerview.widget.RecyclerView;

import android.os.Bundle;

import com.tecwec.stories.databinding.ActivityMainBinding;

import java.util.ArrayList;

public class MainActivity extends AppCompatActivity {
    StatusAdapter statusAdapter;
    ArrayList<user_status> userStatuses;
    ActivityMainBinding binding ;


    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        
        userStatuses=new ArrayList<>();
        statusAdapter=new StatusAdapter(this,userStatuses);
        LinearLayoutManager layoutManager=new LinearLayoutManager(this);
        layoutManager.setOrientation(RecyclerView.HORIZONTAL);
        binding.statusList.setLayoutManager(layoutManager);
        binding.statusList.setAdapter(statusAdapter);
    }
}

File: Android_kotlin/app/src/main/java/com/example/android_kotlin/MainActivity.kt
package com.example.android_kotlin

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button
import android.widget.Toast

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

//        butt.setOnClickListener(){
//            Toast.makeText(this, "this is toast", Toast.LENGTH_SHORT).show();
//        }
    }
}